models/genmodels.go

```diff
@@ -45,6 +61,15 @@ func Generate(packageName, basePath, outputPath string, s spec.Swagger) error {
 		return fmt.Errorf("error generating go-swagger models: %s", err)
 	}
 
+	// Generate our own usercontext.go with type alias to rbac-scoping-utils
+	// The x-user-context extension doesn't reference a swagger definition,
+	// so we don't need to inject/remove it from the spec
+	if hasUserContext {
+		if err := generateUserContextAlias(basePath); err != nil {
+			return fmt.Errorf("error generating UserContext alias: %s", err)
+		}
+	}
```

Comment:
This change checks if a `usercontext` model file should be generated, which should happen if any endpoints need the XUserContext param